packages/utilities/src/memoize.test.ts

```diff
@@ -98,6 +98,60 @@ describe('memoizeFunction', () => {
     expect(func('a')).toEqual('a4');
     expect(func('a')).toEqual('a4');
   });
+
+  it('caches and preserves falsey values other than null and undefined', () => {
+    let returnFalseyValue = true;
+
+    // tslint:disable-next-line:no-any
+    let falseyValue: any = null;
+
+    let callback = (): number | null => {
+      if (returnFalseyValue) { return falseyValue; }
```

Comment:
This formatting (and other places in this file) is also non-standard and should have been fixed by Prettier.

Reply:
There was some issue with the setup. Now Prettier is taking care of formatting.